File: Data Structures & Algorithms/two-integer-sum-ii/submission-10.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {

        vector<int> res;

        for (int i=0; i < numbers.size(); i++) {
            for (int j=i+1; j < numbers.size(); j++) {

                if (numbers[i] + numbers[j] == target) {

                    return {min(i+1, j+1), max(i+1, j+1)};
                }
            }
        }
        return {};
    }
};

File: Data Structures & Algorithms/longest-substring-without-duplicates/submission-10.py
class Solution:
    def lengthOfLongestSubstring(self, s: str) -> int:
        
        l = 0
        r = 0

        best = 0

        while r < len(s):

            while s[r] in s[l:r]:
                l += 1
            
            best = max(best, r - l + 1)

            r += 1

        return best